.github/workflows/test-membership-action.yml

```diff
@@ -0,0 +1,26 @@
+# An action used for testing the "membership" action
+name: Test "membership" action
+on:
+  workflow_dispatch:
+    inputs:
+      GH_TOKEN:
+        required: true
```

Comment:
This repo is public so not sure if there is anyway the token can be exposed. 
May be mask it https://stackoverflow.com/questions/67608874/how-to-mask-workflow-dispatch-inputs-like-secrets?

Reply:
Good spot, I did already think about this but I'm not sure it's possible/necessary.

_Some_ workflow types support a `secret` type for an input, but not `composite` I don't think.
[You can ask GitHub to mask... anything](https://aaron-powell.com/posts/2022-07-14-working-with-add-mask-and-github-actions/), but bearing in mind that it's an input, it could be logged before you even get to execute a line of code.

Practically, anyone calling this script will be supplying a `secret` token - and the logs will be going into that parent action - so it'll already be masked.